postgresql/migrations/000014_lsps2_buy.up.sql

```diff
@@ -34,5 +34,7 @@ CREATE INDEX idx_lsps2_bought_channels_registration_id ON lsps2.bought_channels
 
 CREATE TABLE lsps2.promises (
 	promise varchar PRIMARY KEY,
-	token varchar NOT NULL
-)
+	token varchar NOT NULL,
+	valid_until varchar NOT NULL
```

Comment:
Doesn't adding a not null column require default value?

Reply:
It's always set here:
https://github.com/breez/lspd/pull/130/files#diff-e721fefad0f3dcd1d7884161d3886c91ff99412d1d2cff025e99df0384c7cce8R244

Reply:
Ah.. makes sense